src/dialogs/quick-open/ha-quick-open-dialog.ts

```diff
@@ -0,0 +1,225 @@
+import "../../components/ha-header-bar";
+import "@polymer/paper-input/paper-input";
+import "@material/mwc-list/mwc-list-item";
+import "@material/mwc-list/mwc-list";
+import {
+  css,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+} from "lit-element";
+import { fireEvent } from "../../common/dom/fire_event";
+import "../../components/ha-dialog";
+import { haStyleDialog } from "../../resources/styles";
+import { HomeAssistant, ServiceCallRequest } from "../../types";
+import { PolymerChangedEvent } from "../../polymer-types";
+import { fuzzySequentialMatch } from "../../common/string/sequence_matching";
+import { componentsWithService } from "../../common/config/components_with_service";
+import { domainIcon } from "../../common/entity/domain_icon";
+import { computeDomain } from "../../common/entity/compute_domain";
+import { domainToName } from "../../data/integration";
+import { QuickOpenDialogParams } from "./show-dialog-quick-open";
+import { HassEntity } from "home-assistant-js-websocket";
+import { compare } from "../../common/string/compare";
+import memoizeOne from "memoize-one";
+
+interface CommandItem extends ServiceCallRequest {
+  text: string;
+}
```

Comment:
Linter complained about using {} as a type with the `serviceData?: {}` property.

There are a few solutions to this that don't involve adding to types.ts, but since the command bar will be a place for putting together all kinds of service requests, this seemed like a good opportunity to give more resolution to our request interfaces.